packages/recommend-react/src/__tests__/useRecommendations.test.ts

```diff
@@ -24,11 +30,11 @@ function createMockedRecommendClient() {
 }
 
 describe('useRecommendations', () => {
-  test('gets recommendations', () => {
+  test('gets recommendations', async () => {
```

Comment:
We can apply the same vocabulary in all the other tests as well.

Reply:
✅